### ISSUE TITLE
[APP SUBMITTED]: DAILYSEARCHER :: [a1bf3b3] Missing time zone for network: Viceland (US). Check valid network is set in indexer (theTVDB) before filing issue.

### ISSUE THREAD

**Issue by iamthesq** [SickRage/SickRage#2662]
### INFO
Python Version: **2.7.10 (default, May 23 2015, 09:44:00) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-8-6.2.9200**
Locale: cp1252
Branch: **master**
Commit: SickRage/SickRage@a1bf3b30a9a0d6319c09fb0cf48d2d468dbf4e61
No Log available with ERRORS:
### ERROR
```
DAILYSEARCHER :: [a1bf3b3] Missing time zone for network: Viceland (US). Check valid network is set in indexer (theTVDB) before filing issue.
```
---
_STAFF NOTIFIED_: @SickRage/owners @SickRage/moderators

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by OldskoolOrion** [comment]
### INFO
Python Version: **2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:24:40) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-10-10.0.14986**
Locale: cp1252
Branch: **master**
Commit: SickRage/SickRage@a1bf3b30a9a0d6319c09fb0cf48d2d468dbf4e61
No Log available with ERRORS:
### ERROR
```
Thread-16 :: [a1bf3b3] Missing time zone for network: Viceland (US). Check valid network is set in indexer (theTVDB) before filing issue.
```
---
_STAFF NOTIFIED_: @SickRage/owners @SickRage/moderators

**Comment by OldskoolOrion** [comment]
Nope.. spoke to soon.. it reverts back to Viceland so apparantly that table always gets refreshed / filled with fresh data.. soo.. that means an inconsistency in the TheTVDB.com db ?

**Comment by iamthesq** [comment]
The series I was having trouble with was "Cyberwar."

I logged into TheTVDB to edit the network associated with the show. It seemed to be an invalid network. We'll see if that resolves the issue.

**Comment by OldskoolOrion** [comment]
That's at most a quick fix for you for just that show.. adding Viceland (US) & Viceland (CA) to the list of valid networks is a solution for everybody and all future series those two networks will put out :-)

**Comment by bobbysteel** [comment]
https://github.com/SickRage/sickrage.github.io/pull/215
